apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/converter/db/TransactionModelToEntityConverter.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c)  2018-2020. Apollo Foundation.
```

Comment:
Maybe it is better to update it to 2021?

Reply:
done